models/policy_document.go

```diff
@@ -0,0 +1,81 @@
+package models
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+type PolicyDocument struct {
+	Version   string
+	Statement []statement
+}
+
+type statement struct {
```

Comment:
does aws-sdk-go have a struct for this? nothing wrong with this implementation; just more curious if anything. 
Maybe if we're lucky aws-sdk-go has some nice helpers around this stuff?
```
policy := aws.PolicyDoc{}
policy.Statements.AddAction(...)
```